aten/src/ATen/native/quantized/cpu/qreduction.cpp

```diff
@@ -85,12 +91,8 @@ Tensor& mean_out_quantized_cpu(const Tensor& self,
       self.scalar_type() == kQUInt8 &&
       // QNNPACK currently is only supported for NCHW + dim=(2, 3)
       // Remove these checks after generic version is implemented.
-      self.ndimension() == 4 &&
-      dim.size() == 2 &&
-      dim[0] == 2 &&
-      dim[1] == 3
-     ){
-    result = qnnpack_mean(self, dim);
+      self.ndimension() == 4 && dim.size() == 2 && dim[0] == 2 && dim[1] == 3) {
```

Comment:
I think you may get linter error for this.

Reply:
that's how it got after manual `clang-format` execution (I was burned by formatting check before, so ran it manually)